src/sentry/utils/sdk.py

```diff
@@ -226,6 +226,13 @@ def before_send_transaction(event: Event, _: Hint) -> Event | None:
     ):
         return None
 
+    # This code is added only for debugging purposes, as such, it should be removed once the investigation is done.
+    if event.get("transaction") in {
+        "sentry.dynamic_sampling.boost_low_volume_projects_of_org",
+        "sentry.dynamic_sampling.tasks.boost_low_volume_projects",
+    }:
```

Comment:
Regarding performance, both transactions happen every 10 minutes, so it should not be a concern.

Reply:
Yes, this looks OK!